PyDSS/ResultData.py

```diff
@@ -86,10 +86,11 @@ def _create_element_list(self, objs, properties):
             for name, obj in objs.items():
                 if not obj.IsValidAttribute(property_name):
                     raise InvalidParameter(f"{element_class} / {property_name} {name} cannot be exported")
+                #TODO: Will commenting this be an issue ever Dan?
```

Comment:
This code looks good now. You can delete the TODO line.

Reply:
Will do that

Reply:
done